modules/schematics/src/action/index.spec.ts

```diff
@@ -69,6 +69,32 @@ describe('Action Schematic', () => {
     expect(fileContent).toMatch(/export enum FooActionTypes/);
   });
 
+  it('should create a class named "LoadFoos"', () => {
+    const tree = schematicRunner.runSchematic(
+      'action',
+      defaultOptions,
+      appTree
+    );
+    const fileContent = tree.readContent(
+      `${projectPath}/src/app/foo.actions.ts`
+    );
+
+    expect(fileContent).toMatch(/export class LoadFoos implements Action/);
+  });
+
+  it('should create a type named "FooActions"', () => {
```

Comment:
This should be more like `should create the union type based on the provided name`